src/app/App.test.tsx

```diff
@@ -97,8 +97,10 @@ describe("App", () => {
 			expect(passwordInput()).toHaveValue("password");
 		});
 
+		const profile = env.profiles().findById("cba050f1-880f-45f0-9af9-cfe48f406052");
+
 		const verifyPasswordMock = vi.spyOn(Bcrypt, "verify").mockReturnValue(true);
-		const memoryPasswordMock = vi.spyOn(env.profiles().last().password(), "get").mockImplementation(() => {
```

Comment:
Since I added a new profile for Mainsail the given profile is no longer the `last` one